app/components/elements/AreaSummary/index.jsx

```diff
@@ -34,10 +34,10 @@ class AreaSummary extends React.Component {
     let { selectedYear } = this.state;
 
     return (<div className="content">
-      <div  className={ classname(
-          location.href.includes("reach") ? 'reach' : 'impact',
-           'c-area-summary')}>
-
+      <div className={classname(
+        location.href.includes("reach") ? 'reach' : 'impact',
+        'c-area-summary')}>
+        <div className="area-summary-wrapper">
           <span dangerouslySetInnerHTML={{ __html: title }} />
```

Comment:
Review if you are using HTML here. Also, I don't like the idea of putting a title inside a `span` tag.